src/main/target/MICROSCISKY/target.h

```diff
@@ -33,9 +33,7 @@
 
 #define INVERTER_PIN_UART2      PB2 // PB2 (BOOT1) abused as inverter select GPIO
 
-#define USE_EXTI
 #define MAG_INT_EXTI            PC14
-#define USE_MPU_DATA_READY_SIGNAL
 #define USE_MAG_DATA_READY_SIGNAL
 
```

Comment:
😮 MAG also uses similar defines. Outside of the scope of this PR.

Reply:
The same story with `USE_MAG_DATA_READY_SIGNAL` probably. We can add it to this PR.

Reply:
Only one instance that I can see so it can go.